internal/cmd/schema-registry/command_exporter_create.go

```diff
@@ -48,6 +48,11 @@ func (c *command) newExporterCreateCommand(cfg *v1.Config) *cobra.Command {
 	}
 	pcmd.AddOutputFlag(cmd)
 
+	// Deprecated
+	cmd.Flags().String("config-file", "", "Exporter configuration file.")
```

Comment:
may use the description `"Name of a file with additional topic configuration. Each property should be on its own line with the format: key=value."` here as well to be more informative

Reply:
This flag is deprecated (hidden), so users won't see this description. I just copied the original declaration from above down here.